<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="GS-5177" name="login_suite" tests="2" failures="1" timestamp="24 Sep 2021 08:27:43 GMT" time="9.087" errors="0">
  <testcase name="login2" time="0.021" classname="testcases.LoginTest"/>
  <testcase name="login1" time="0.052" classname="testcases.LoginTest">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at testcases.LoginTest.login1(LoginTest.java:17)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
... Removed 14 stack frames]]>
    </failure>
  </testcase> <!-- login1 -->
</testsuite> <!-- login_suite -->
